libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/PasscodeActivity.java

```diff
@@ -642,4 +647,11 @@ private void clearUi() {
         biometricBox.setVisibility(View.GONE);
         fingerImage.setVisibility(View.GONE);
     }
+
+    private void hideKeyboard() {
+        InputMethodManager imm = (InputMethodManager) this.getSystemService(Activity.INPUT_METHOD_SERVICE);
+        if (this.passcodeField != null) {
```

Comment:
`this` is implicit. You don't need to explicitly add it.

Reply:
Whoops.  Made the change locally, I will commit it when UX feedback comes back.